docs/rewards/rewards-troubleshooting.mdx

```diff
@@ -145,12 +158,25 @@ Please make sure to check the troubleshooting guides depending on your weather s
 
 ## Location Not Verified
 
-It seems that your location is different from the one you set while claiming your station.
-Please verify your location through the mobile app (Station details -> Settings -> Edit Location)
+It appears that the location of your weather station is different from the one you had originally set while claiming it. 
+Please verify your location through the mobile app. (See the screenshots below for guidance)
```

Comment:
"(See the screenshots below for guidance)" is not needed IMO as the screenshots are right below the text so the user will see them either way.